### ISSUE TITLE
1.20.1 | Server crash upon wearing CT-edited armor(?)

### ISSUE THREAD

**Issue by skall0900** [Momo-Studios/Cold-Sweat-Rework-Forge#305]
Hi! I've been changing some attributes of one armor, but sometimes, while testing in survival mode, when putting on the armor, the server crashed. Keyword *sometimes*, I'm not really sure. If its not Cold Sweat's fault, I'll make an issue on CT issue tracker instead.

Version: Minecraft1.20.1, Forge 47.2.30

Affected mods:
Suspected on crashlog:
Crafttweaker (ver. 14.0.40), [modrinth page](https://modrinth.com/mod/crafttweaker)
Cold Sweat (ver. 2.3-b03e), [curseforge page just in case](https://www.curseforge.com/minecraft/mc-mods/cold-sweat)

Other used mods:
Pufferfish's Attributes (ver. 0.2.0; used for new attributes themself), [modrinth page](https://modrinth.com/mod/attributes)
Lucky's Armory (ver. 0.3.0; edited armor, to be exact, Heavy Iron Armor (fullset)), [curseforge page](https://modrinth.com/mod/attributes)

Logs:
[crash report](https://pastebin.com/bM0zyJek)
[crafttweaker script if it helps](https://pastebin.com/gLf0aXjz)
[latest.log](https://pastebin.com/3nnUphky) (pretty much edited to the point of world start to avoid 512kb limit)

**Comment by MikulDev** [comment]
[ColdSweat-2.3-b03f.zip](https://github.com/Momo-Softworks/Cold-Sweat/files/15489156/ColdSweat-2.3-b03f.zip)
Here's a quick fix I made that I think might solve the issue. Let me know if it works

**Comment by skall0900** [comment]
Hi! Sorry for late reply, different timezones.
Tested it through, still crashes, ~~but now it seems it crashes when *hovering over armor*, while wearing it.~~ looks like it crashes by simply wearing said armor.
[Crash log](https://pastebin.com/5vwavXyb)
[Latest log](https://pastebin.com/zML4E8tT)

**Comment by MikulDev** [comment]
Upon closer inspection, I think this might be a CraftTweaker bug. Try posting this issue to their page. Hopefully they can shed some light on this.

**Comment by skall0900** [comment]
Hm, thats odd. I tested the armor on isolated build (from all 100+ mods), it worked just fine, then I copied the CS config files to the main build, and now it works fine, without crashing. Thus, config reset sealed the deal.

Seems the issue was in **old configs**, as I never changed them while updating beta versions of CS.

Still, thanks for some quick help! I'll close the issue. gl with mod development, it is amazing :)
